src/components/AccommodationDetails/AccommodationDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import {AccommodationDetailsWrapper, AccommodationDetailsInner, Image, RightContent, HotelData, HotelName, Stars, Star, Subtitle, CancelationDetails, Cancelation, Calendar, Description, LeftContent, PropertyInfo, LeftContentInner, Text} from './AccommodationDetailsStyle';
+import BackgroundImage from '../../assets/accommodationdetails.svg';
+import Navigation from '../Navigation/Navigation';
+import Button from '../Button/Button';
+import Footer from '../Footer/Footer';
+import StarImage from '../../assets/star.png';
+import accommodationDetailsMock from '../../lib/mock/accommodationdetails';
+
+const AccommodationDetails = () => {
+    return (
+        <>
+            <Navigation />
+            <AccommodationDetailsWrapper>
+                <Image src={BackgroundImage} />
+
+                {accommodationDetailsMock.map((accommodation, index) => (
+                <AccommodationDetailsInner>
+                    <LeftContent>
+                        <HotelData>
+                            <HotelName>{accommodation.title}</HotelName>
+                            <Stars>
+                                {[...Array(accommodation.categorization)].map((index) => ( 
+                                    <Star key={index} src={StarImage}/>
+                                ))}
```

Comment:
Ja bih ovo malo prepravila, na sljedeći način:
```
                            <Stars>
                                {getStars(accommodation.categorization).map((i) => ( 
                                    <Star key={i} src={StarImage}/>
                                ))}
                            </Stars>
```

Pri tome getStarts je funkcija koju možeš kreirati prije return, na sljedeći način:
```
    const getStars = (numberOfStars: number) => {
        return Array.from(Array(numberOfStars).keys());
    }
```